notification-server/Cargo.toml

```diff
@@ -2,7 +2,7 @@
 authors = ["Concordium AG developers@concordium.com"]
 edition = "2021"
 name = "notification-server"
-version = "0.1.0"
+version = "0.1.4"
```

Comment:
How come you jump from .0 to .4?

Reply:
Had to cancel one related to some dependency stuff. 

Do you want the exact details for each individual deployment?

Reply:
Incrementing the version (without jumps) should be our goal for each PR. 

To achieve this, I suggest:
- Deployments and builds pushed to docker hubs should be from main (of merged/reviewed code).
- Since we are not yet hosting the `notification-service`, do we already need to push the image?
- If we want to test changes to the CI pipeline before merging it to main, we could comment out the line that pushes the docker image and run the CI pipeline in this PR, and then comment in the line again if successful.

Reply:
I am not testing the CI pipeline. I am making the actual deployment code. 

And yes that would be ideal.